### ISSUE TITLE
val loss=0,pixel acc=1.000，miou=0.5000，始终不变,应该怎么解决

### ISSUE THREAD

**Issue by dqb315421** [yuliangguo/Pytorch_Generalized_3D_Lane_Detection#14]

**Comment by Snoooooooooopy** [comment]
Hello, I also encountered this problem, did you solve this problem later?
